### PR TITLE
Fix request hook

### PR DESCRIPTION
**Pull request by taobun**
Fixed: #XXXX

## Implementation details
- Create hook in app instead of cmd
- Send hook to oracle module using in Querier

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: cmd/bandd/cmd/root.go [hunk 1]
```diff
@@ -242,50 +249,10 @@ func (ac appCreator) newApp(logger log.Logger, db dbm.DB, traceStore io.Writer,
...
-			if err != nil {
-				panic(err)
-			}
-			oracleIDs = append(oracleIDs, oracletypes.OracleScriptID(oracleID))
-		}
-		bandApp.AddHook(
-			price.NewHook(bandApp.AppCodec(), bandApp.OracleKeeper, oracleIDs,
-				filepath.Join(cast.ToString(appOpts.Get(cli.HomeFlag)), "prices"),
-				defaultAskCount, defaultMinCount))
+
```
Comment (Benzbeeb) [inline cmd/bandd/cmd/root.go:255]: remove empty `if` clause

File: x/oracle/keeper/grpc_query.go [hunk 2]
```diff
@@ -240,18 +279,30 @@ func (k Querier) Params(c context.Context, req *types.QueryParamsRequest) (*type
...
 
 // RequestPrice queries the latest price on standard price reference oracle
 // script.
-func (k Querier) RequestPrice(c context.Context, req *types.QueryRequestPriceRequest) (*types.QueryRequestPriceResponse, error) {
-	return nil, status.Errorf(codes.Unimplemented, "This feature can be taken from extra/rest branch")
+func (k Querier) RequestPrice(
+	c context.Context,
+	req *types.QueryRequestPriceRequest,
+) (*types.QueryRequestPriceResponse, error) {
+	fmt.Println("Request Price", req)
```
Comment (songwongtp) [inline x/oracle/keeper/grpc_query.go:296]: Remove debug

**Review by Benzbeeb — APPROVED** [review]
LGTM
Dog LGTM Keyboard

**Review by songwongtp — APPROVED** [review]
https://media.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif

**Review by satawatnack — APPROVED** [review]

**Review by RogerKSI — APPROVED** [review]

**Pull request merged**